### PR TITLE
#3139 ETO preview cards success state design change

### PR DESCRIPTION
**Pull request by desfero**

### CODE REVIEW

File: app/components/dashboard/eto-list/EtoList.tsx [hunk 1]
```diff
@@ -39,16 +39,15 @@ const EtoListThumbnails: React.FunctionComponent<TListProps & TExternalProps> =
...
+    {mockedEtos.map(eto => (
+      <EtoOverviewThumbnail
+        mockedEto={eto}
+        key={eto.id}
+        shouldOpenInNewWindow={shouldOpenInNewWindow}
+      />
+    ))}
 
-    {etos.length < 4 && <EtosComingSoonThumbnail />}
+    {etos.length + mockedEtos.length < 4 && <EtosComingSoonThumbnail />}
```
Comment (Moejoe90) [inline app/components/dashboard/eto-list/EtoList.tsx]: Hmmm the IMO will still remain in prod as we don't have `mockedEtos` on Prod or am I wrong?

File: app/components/eto/overview/EtoOverviewThumbnail/EtoStatusManager.tsx [hunk 2]
```diff
@@ -26,21 +28,24 @@ interface IExternalProps {
...
-            inputFormat={ENumberInputFormat.ULPS}
-            moneyFormat={ECurrency.EUR}
-            outputFormat={ENumberOutputFormat.FULL}
-          />
-        ),
-      }}
-    />
-  </p>
+  <div className={styles.successfulInfo}>
+    <p className={cn(styles.successfulInfoText)}>Successful fundraising</p>
```
Comment (Moejoe90) [inline app/components/eto/overview/EtoOverviewThumbnail/EtoStatusManager.tsx]: Hmmm we need a translation for `Successful fundraising`

Comment (desfero) [reply app/components/eto/overview/EtoOverviewThumbnail/EtoStatusManager.tsx]: ahh, i added the translations string but forgot to move it here

File: app/components/dashboard/eto-list/EtoList.tsx (hunk 1 above)
Comment (Moejoe90) [reply app/components/dashboard/eto-list/EtoList.tsx]: NVM I understood wrong the requirements

**Pull request merged**